### PR TITLE
WIP: Operation reports output to a PPT

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: app/service/rest_svc.py [hunk 1]
```diff
@@ -262,3 +264,6 @@ def _get_stopping_conditions(data):
         contact_svc.sleep_min = sleep_min
         contact_svc.sleep_max = sleep_max
         contact_svc.watchdog = watchdog
+
+    async def _generate_ppt_report(self, op):
```
Comment (privateducky) [inline app/service/rest_svc.py]: check redbook if you'd like some sample code for this..

File: app/service/rest_svc.py [hunk 2]
```diff
@@ -3,12 +3,15 @@
 import glob
 import os
 import pathlib
+import pptx
 import uuid
 import yaml
 
 from collections import defaultdict
 from datetime import time
```
Comment (wbooth) [inline app/service/rest_svc.py]: https://www.python.org/dev/peps/pep-0008/#imports can you re arrange the imports?

Comment (khyberspache) [reply app/service/rest_svc.py]: This branch is way out of date & I haven't touched it in a while... but yes

**Comment by khyberspache** [comment]
Closing for now, no demand signal right now.  Should be an easy add in the future...

**Pull request closed without merging**